hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkTableServiceClient.java

```diff
@@ -243,7 +169,31 @@ public HoodieWriteMetadata<List<WriteStatus>> cluster(String clusteringInstant,
   }
 
   @Override
-  protected HoodieTable<?, ?, ?, ?> createTable(HoodieWriteConfig config, Configuration hadoopConf) {
+  protected void validateClusteringCommit(HoodieWriteMetadata<List<WriteStatus>> clusteringMetadata, String clusteringCommitTime, HoodieTable table) {
```

Comment:
clustering is not supported in flink. we should just throw exception here

Reply:
Updated